docs/index.md

```diff
@@ -137,51 +68,51 @@ So *pydantic* uses some cool new language features, but why should I actually go
 Hundreds of organisations and packages are using *pydantic*, including:
 
 [FastAPI](https://fastapi.tiangolo.com/)
-: a high performance API framework, easy to learn,
+: A high performance API framework, easy to learn,
   fast to code and ready for production, based on *pydantic* and Starlette.
 
 [Project Jupyter](https://jupyter.org/)
-: developers of the Jupyter notebook are using *pydantic*
+: Developers of the Jupyter notebook are using *pydantic*
```

Comment:
Let's talk about which packages to mention here - this list is very old and definitely needs to be updated.

Reply:
Making this a separate task to follow up on